File: tests/fixtures/java-scaffold/src/main/java/com/example/service/CustomerService.java
package com.example.service;

import java.util.ArrayList;
import java.util.List;

public class CustomerService {
    private final List<String> customers = new ArrayList<>();

    public CustomerService() {
        customers.add("Ada");
        customers.add("Grace");
    }

    public List<String> findAll() {
        return new ArrayList<>(customers);
    }

    public boolean exists(String name) {
        return customers.stream().anyMatch(candidate -> candidate.equalsIgnoreCase(name));
    }

    public int count() {
        return customers.size();
    }
}

File: languages/javascript/tests/fixtures/python-scaffold/src/services/customer_service.py
class CustomerService:
    """Simple service for demonstration purposes."""

    def __init__(self) -> None:
        self._customers = ["Ada", "Grace"]

    def find_all(self) -> list[str]:
        return list(self._customers)

    def add_customer(self, name: str) -> None:
        self._customers.append(name)


def helper() -> bool:
    return True
